pkg/proxy/proxy.go

```diff
@@ -40,12 +40,13 @@ const (
 	appLabel            = "app"
 	releaseLabel        = "release"
 
-	exposeAnnotation        = "fabric8.io/expose"
-	exposeIngressAnnotation = "fabric8.io/ingress.annotations"
-	ingressNameAnnotation   = "fabric8.io/ingress.name"
-	ingressClassAnnotations = "kubernetes.io/ingress.class"
-	certManagerAnnotation   = "certmanager.k8s.io/issuer"
-	ingressClass            = "nginx"
+	exposeAnnotation         = "fabric8.io/expose"
+	exposeIngressAnnotation  = "fabric8.io/ingress.annotations"
+	ingressNameAnnotation    = "fabric8.io/ingress.name"
+	ingressClassAnnotations  = "kubernetes.io/ingress.class"
+	oldCertManagerAnnotation = "certmanager.k8s.io/issuer"
+	certManagerAnnotation    = "certmanager.io/issuer"
```

Comment:
Should be `cert-manager.io/issuer` - see https://cert-manager.io/docs/usage/ingress/#supported-annotations

Reply:
okay